script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -1045,6 +1048,8 @@ The girl lashes out over and over again in a blind rage,
 // Lotta のに here and next sentence.
 // Can't immediately come up with something that keeps them all perfectly the same,
 // but this has a pretty readable pattern at least
+// If you had just?
+// Restructure to try and better encode the なら / のに
 "Why? Why?!
  You could have kept up the act forever!
  You could have stayed the way you were, stayed inhuman, and even I would have forgiven you!
```

Comment:
>Why? Why?! You could have just kept up the act! I could forgive when you stayed an inhuman monster!

>You could have just stayed a machine that did nothing but hunt vampires and fight! I could admire you from the bottom of my heart then!

Tried to keep it to two distinct pattern instead of it being the same for all for (which, incidentally is kinda like the JP, where it's のに at the end for all of them, but 良かったのに specifically for the first two.
You could - I would